files/ubuntu-18.04/configure_nw_interface.sh

```diff
@@ -57,4 +58,22 @@ $STATIC_IP_CONFIG
       routing-policy:
         - from: ${DEVICE_IP_ADDRESS}
           table: ${ROUTE_TABLE}
-EOF
\ No newline at end of file
+EOF
+
+# Hook to delete automatic routes to subnet automatically created from kernel.
+# This is needed in Ubuntu 18 because they are generated in the reverse order in which the network interfaces are
+# attached and this prevents the primary Network Interface from being selected to communicate in the subnet
+FILE="/etc/networkd-dispatcher/routable.d/cleanup-routes.sh"
+
+if [ ! -f "$FILE" ]; then
+/bin/cat <<EOF >${FILE}
+#!/bin/sh
+
+logger -t parallelcluster "Removing Automatic route for Interface: \${IFACE}"
```

Comment:
where `IFACE` is defined?

Reply:
It's an environment variable passed from [networkd-dispatcher](https://gitlab.com/craftyguy/networkd-dispatcher).